spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
+Capybara.default_driver = :rack_test
```

Comment:
Sure rack test is fine for now, if we ever need a driver that can execute javascript poltergeist is my preferred driver